src/openrct2/world/Map.cpp

```diff
@@ -1741,7 +1741,7 @@ static void clear_element_at(const CoordsXY& loc, TileElement** elementPtr)
                     break;
             }
             auto parkEntranceRemoveAction = ParkEntranceRemoveAction(CoordsXYZ{ seqLoc, element->GetBaseZ() });
-            auto result = GameActions::Execute(&parkEntranceRemoveAction);
+            auto result = GameActions::ExecuteNested(&parkEntranceRemoveAction);
```

Comment:
Have you made sure that clear_element_at is not called from outside a game action? Otherwise this will break current code.

Reply:
Its used by `map_init`, `decrease_map_size` (this PR), `mapgen_generate_blank`, `mapgen_generate`. All of which are single player (apart from obviously this PR). Its safe to perform a nested from singleplayer instances although it isn't pretty. Alternative would be to add a new mode to the `ClearAction` that performs a full wipe of a tile including park entrances and peep spawns. This can be left for a follow on PR though. An issue is to be created for this though so its not forgotten otherwise some person in the future might call this action at the wrong time.